frontend/views/utils/vError.js

```diff
@@ -0,0 +1,34 @@
+import Vue from 'vue'
+
+// Register a global custom directive called `v-error`
+// to automatically display vuelidate error messages
+
+Vue.directive('error', {
+  inserted (el, binding, vnode) {
+    if (!binding.value) {
+      throw new Error(`v-error: missing argument on ${el.outerHTML}`)
+    }
+    if (!vnode.context.$v.form[binding.value]) {
+      throw new Error(`v-error: vuelidate doesn't have validation for ${binding.value}`)
+    }
+    const pErr = document.createElement('p')
```

Comment:
change it to a `span`. most of the times this error will be inside a `label` element and block elements aren't allowed as children